files.xml

```diff
@@ -53,20 +53,20 @@
 		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-implementation.c" />
 		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-linear-gradient.c" />
 		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-matrix.c" />
-		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-mips.c" />
-		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-mmx.c" if="windows || mac || linux" unless="rpi"/>
+		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-mips.c" unless="rpi || HXCPP_ARM64" />
+		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-mmx.c" if="windows || mac || linux" unless="rpi || HXCPP_ARM64"/>
 		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-noop.c" />
 		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-ppc.c" />
 		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-radial-gradient.c" />
 		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-region16.c" />
 		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-region32.c" />
 		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-solid-fill.c" />
-		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-sse2.c" if="windows || mac || linux" unless="rpi"/>
-		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-ssse3.c" if="windows || mac || linux" unless="rpi"/>
+		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-sse2.c" if="windows || mac || linux" unless="rpi || HXCPP_ARM64"/>
+		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-ssse3.c" if="windows || mac || linux" unless="rpi || HXCPP_ARM64"/>
 		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-timer.c" />
 		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-trap.c" />
 		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-utils.c" />
-		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-x86.c" />
+		<file name="${NATIVE_TOOLKIT_PATH}/pixman/src/pixman-x86.c" if="windows || mac || linux" unless="rpi || HXCPP_ARM64"/>
```

Comment:
Not including `pixman-x86.c` and `pixman-mips.c` causes linking errors:

    "__pixman_mips_get_implementations", referenced from:
        __pixman_choose_implementation in 2ec37e17_pixman-implementation.o
    "__pixman_x86_get_implementations", referenced from:
        __pixman_choose_implementation in 2ec37e17_pixman-implementation.o


Both `pixman-x86.c` and `pixman-mips.c` should be built also on arm64 platforms, otherwise lime.ndll will not link properly. Instead, the config.h needs to be adjusted to fix compilation. See:
https://github.com/soccertutor/lime/commit/2419e7e250f8c18d2d912ce77cb626d9fb0f6973

Reply:
Thanks for taking up this issue ^^